r2dbc-mysql/src/main/java/io/asyncer/r2dbc/mysql/MySqlConnectionConfiguration.java

```diff
@@ -1207,6 +1218,20 @@ public Builder metrics(boolean enabled) {
             return this;
         }
 
+        /**
+         * option to whether the driver should interpret MySQL's TINYINT(1) as a BIT type.
+         * When enabled, TINYINT(1) columns (both SIGNED and UNSIGNED) will be treated as
+         * {@link Boolean} by default.
+         *
+         * @param tinyInt1isBit {@code true} to treat TINYINT(1) as BIT
+         * @return this {@link Builder}
+         * @since 1.4.0
+         */
```

Comment:
```suggestion
        /**
         * Option to whether the driver should interpret MySQL's TINYINT(1) as a BIT type.
         * When enabled, TINYINT(1) columns (both SIGNED and UNSIGNED) will be treated as
         * BIT by default.
         *
         * @param tinyInt1isBit {@code true} to treat TINYINT(1) as BIT
         * @return this {@link Builder}
         * @since 1.4.0
         */
```